com.unity.ml-agents/Runtime/Grpc/GrpcExtensions.cs

```diff
@@ -6,6 +6,9 @@
 using MLAgents.CommunicatorObjects;
 using MLAgents.Sensor;
 using UnityEngine;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Unity.ML-Agents.Editor")]
```

Comment:
Makes the assembly that this file is in internally visible to the Editor assembly.